editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -7657,6 +7838,9 @@ Node3DEditor::Node3DEditor() {
 	// Drag and drop support;
 	preview_node = memnew(Node3D);
 	preview_bounds = AABB();
+	preview_material = Ref<Material>();
+	preview_reset_material = Ref<Material>();
+	preview_material_target = ObjectID();
```

Comment:
You don't need to initialize Godot types (seems like the code above also does this unnecessarily).